lms/djangoapps/certificates/models.py

```diff
@@ -95,7 +95,9 @@ class CertificateStatuses(object):
     readable_statuses = {
         downloadable: "already received",
         notpassing: "didn't receive",
-        error: "error states"
+        error: "error states",
+        audit_passing: "verified but audit passing states",
```

Comment:
this isn't really true though. the "verified but" part is something we added in logic that lives elsewhere